### DISCUSSION TITLE
libustream-openssl can not compile

### DISCUSSION THREAD

**Discussion by ilxp** [immortalwrt/immortalwrt#973] [Q&A]
### Describe the bug

the error logo is：
 * check_data_file_clashes: Package libustream-openssl20201210 wants to install file /home/runner/work/immortalwrt-new/workspace/openwrt/build_dir/target-x86_64_musl/root-x86/lib/libustream-ssl.so
	But that file is already provided by package  * libustream-mbedtls20201210
 * opkg_install_cmd: Cannot install package libustream-openssl20201210.

我在config里面已经做了
# CONFIG_PACKAGE_libustream-mbedtls is not set
CONFIG_PACKAGE_libustream-openssl=y
# CONFIG_PACKAGE_libustream-wolfssl is not set

但还是报错。不知道是哪个插件在用libustream-openssl。如何破解呢

### ImmortalWrt version

2102/2203/snapshots

### ImmortalWrt target/subtarget

x86

### Device

64

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
我们默认使用 libustream-openssl，应该首选。
在 menuconfig 按 `/` 搜索 `libustream-mbedtls` 看被谁依赖了

**Comment by ilxp** [comment]
查明了，是luci-app-unblockneteasemusic的：
@(PACKAGE_libustream-mbedtls||PACKAGE_libustream-openssl||PACKAGE_libustream-wolfssl)

**Comment by ilxp** [comment]
LUCI_TITLE:=LuCI support for UnblockNeteaseMusic (JavaScript)
LUCI_DEPENDS:=+dnsmasq-full +node \
	+PACKAGE_firewall:ipset \
	@(PACKAGE_libustream-mbedtls||PACKAGE_libustream-openssl||PACKAGE_libustream-wolfssl)
LUCI_PKGARCH:=all

PKG_NAME:=luci-app-unblockneteasemusic
PKG_VERSION:=3.2
PKG_RELEASE:=2

PKG_MAINTAINER:=Tianling Shen <cnsztl@immortalwrt.org>

大佬，能否将@(PACKAGE_libustream-mbedtls||PACKAGE_libustream-openssl||PACKAGE_libustream-wolfssl)，调整下排序啊

**Comment by 1715173329** [comment]
目前强制依赖 mbedtls 的也就 luci-ssl 而已，这个包也不做任何事情，去除即可

**Comment by ilxp** [comment]
ok，收到。谢谢大佬
